_data/glossary/kubelet.yaml

```diff
@@ -5,6 +5,6 @@ tags:
 - fundamental
 - core-object
 short-description: >
-  An agent that runs on each node.
+  An agent that runs on each node in the cluster. It makes sure that containers are running in a pod.
 long-description: >
   The kubelet takes a set of PodSpecs that are provided through various mechanisms and ensures that the containers described in those PodSpecs are running and healthy. The kubelet doesn’t manage containers which were not created by Kubernetes.
```

Comment:
I was expecting that the following key kubelet functions mentioned *briefly*:
- container lifecycle management by interacting with container runtimes
- node resource management by abstracting OS level details
- network and storage provisioning for containers, and
- container monitoring regarding health and performance metrics, etc.

Reply:
@tengqm hi, thanks for your comments :-). Since we are trying to keep the text in limit even with long description, I didn't add it earlier. I have provided a link in the glossary which I believe covers the description you have mentioned. If it's needed to add it, I can add it in a separate PR. Thanks!!